ui/imports/utils/Utils.qml

```diff
@@ -511,10 +511,9 @@ QtObject {
     /* Validation section end */
 
     function getContactDetailsAsJson(publicKey, getVerificationRequest=true) {
-        let jsonObj = mainModuleInst.getContactDetailsAsJson(publicKey, getVerificationRequest)
         try {
-            let obj = JSON.parse(jsonObj)
-            return obj
+            const jsonObj = mainModuleInst.getContactDetailsAsJson(publicKey, getVerificationRequest)
```

Comment:
@igor-sirotin I am wondering what fixes this issue cause I see the code is the same as it was except the old line 514 is in `try` block now?

Reply:
In that case would be better to check `!!mainModuleInst`? Or remove message in the line `520` cause it will be displayed every time even in case we don't get to json parse error.